examples/distribution-openshift/rendered_manifests/securityContextConstraints.yaml

```diff
@@ -16,6 +16,8 @@ metadata:
     heritage: Helm
 users:
 - system:serviceaccount:default:default-splunk-otel-collector
+- system:serviceaccount:default:default
+- system:serviceaccount:default:default
```

Comment:
Extra line?

Reply:
Yes! I was not sure because this propped up when I rant "make render'